packages/blade/src/components/Indicator/Indicator.tsx

```diff
@@ -86,16 +85,24 @@ const Indicator = ({
     ...(isWeb && { role: 'status' }),
   });
 
+  const textLabel =
+    size === 'large' && children ? (
+      <Heading contrast="low" type="subtle" weight="regular" size={dimensions.textSize}>
+        {children}
+      </Heading>
+    ) : (
+      <Text contrast="low" type="subtle" size={dimensions.textSize}>
+        {children}
+      </Text>
+    );
+
   return (
     <Box display="flex" flexDirection="row" alignItems="center" {...a11yProps}>
-      <Svg width={dimensions.svgSize} height={dimensions.svgSize} viewBox="0 0 12 12" fill="none">
-        <Circle cx="6" cy="6" r="5.5" fill={fillColor} stroke={strokeColor} />
+      <Svg width={dimensions.svgSize} height={dimensions.svgSize} viewBox="0 0 10 10" fill="none">
```

Comment:
we use viewbox as 24 24 everywhere else. i hope this is intentional here?

Reply:
Yes, intentional. This is from the indicator figma.

Reply:
I think only the icons components are using 24 24 viewbox, checked that other situational icons (like checkbox) aren't using 24 24.